src/graphql/schema.js

```diff
@@ -32,6 +32,9 @@ export type ObjectId = string;
 //   - Connections are supported as object fields, but arbitrary lists
 //     are not.
 //
+// Primitive and enum fields on an object type may optionally be
```

Comment:
Why is it optional?

Reply:
Good question.

This was a slightly easier change to make: it means that I didn’t have
to update all the schemata in test code and such.

There’s something to be said for not requiring people to annotate their
schemata if they don’t care about Flow types. But maybe that’s not a
good argument. For instance, if the annotations are required, then the
Mirror module can store some things more efficiently (it wouldn’t have
to JSON-encode values, because we could represent booleans as SQL `0` or
`1` instead of `'true'` and `'false'`, because we could tell at
extraction time what the type of the result should be.)

I’d be happy to make it required in a future commit.